tests/integration/test_report.go

```diff
@@ -71,7 +72,7 @@ func generateReport() {
 				return nil
 			}
 
-			_, err1 := copyReport(path, fmt.Sprintf("%s/report.html", artifactsDir))
+			_, err1 := copyReport(path, fmt.Sprintf("%s/%s-report.html", artifactsDir, ts.Name()))
```

Comment:
```suggestion
			_, err1 := copyReport(path, fmt.Sprintf("%s/report-%s.html", artifactsDir, ts.Name()))
```
The html Prow lens expects the format to be report*.html